### ISSUE TITLE
Multiline strings format in TOML

### ISSUE THREAD

**Issue by Christian-Madlansacay** [TheElectronWill/night-config#110]
Hey there,

I am trying to create a multiline string and I create it like this:
```java
spec.define("test", """
Line 1
Line 2
Line 3""");
```

But when it loads it in the TOML file, it is formatted like this:
```toml
test = "Line 1\nLine 2\nLine 3
```

I'd like it to be formatted like this in the TOML file, though:
```toml
test = """
Line 1
Line 2
Line 3"""
```

If there is a way I can make it so that it can be formatted the way I'd like it, that'd be helpful. Thanks

**Comment by TheElectronWill** [comment]
Hey!

There isn't a way to configure the writer for multiline strings... yet. I'll think about it for the next release :)

**Comment by Christian-Madlansacay** [comment]
I really hope that this feature gets added. Thank you!

**Comment by TheElectronWill** [comment]
I'll keep it open and close it when it's added

**Comment by TheElectronWill** [comment]
Implemented in 26f7ff42f70f32be90a478a6fb99613cc437bd51 :tada:
